skxray/correlation.py

```diff
@@ -367,3 +344,337 @@ def extract_label_indices(labels):
     label_mask = labels[labels > 0]
 
     return label_mask, pixel_list
+
+
+def two_time_corr(num_levels, num_bufs, labels, images):
```

Comment:
I'm not sure the `_corr` is needed, as the module where this function resides is called `correlation`.  I think that the import would look nice and be just as descriptive if the `_corr` was dropped.  e.g.,  
```python
from skxray.correlation import two_time
```

Thoughts?